src/VectorizeLoops.cpp

```diff
@@ -1022,6 +1431,7 @@ class VectorizeLoops : public IRMutator {
             // Replace the var with a ramp within the body
             Expr for_var = Variable::make(Int(32), for_loop->name);
             Expr replacement = Ramp::make(for_loop->min, 1, extent->value);
+            stmt = for_loop->body;
```

Comment:
...is this a pre-existing bug?

Reply:
No, I think this additional line is a no-op, because the next line doesn't use stmt and just clobbers it. I'll delete it.